src/iop/agx.c

```diff
@@ -2039,8 +2039,9 @@ static GtkWidget *_add_primaries_box(dt_iop_module_t *self)
 
   GtkWidget *primaries_box = gtk_box_new(GTK_ORIENTATION_VERTICAL, DT_BAUHAUS_SPACE);
   self->widget = primaries_box;
+  dt_iop_module_t *section = DT_IOP_SECTION_FOR_PARAMS(self, _("primaries"));
```

Comment:
This almost made me consider allowing `DT_IOP_SECTION_FOR_PARAMS` to allow nesting (so you could create a primaries\blue or primaries\before section and not have too much crowded in one group. But it wouldn't add too much value.